### ISSUE TITLE
Allow the user to navigate to the "all results" page by hitting enter on the keyboard. 

### ISSUE THREAD

**Issue by ncameronbritt** [openshift/origin-web-console#1949]
Hitting enter in the catalog search field should be the same as clicking "view all results"

Desired behavior is documented here:
http://openshift.github.io/openshift-origin-design/web-console/1-homepage/search-filter#search-catalog

![screen shot 2017-08-17 at 3 26 06 pm](https://user-images.githubusercontent.com/8974593/29429970-9b1d54da-8360-11e7-8b81-ab1184a571bb.png)
